Emby.Drawing/Common/ImageHeader.cs

```diff
@@ -50,12 +50,15 @@ public static class ImageHeader
         /// <exception cref="ArgumentException">The image was of an unrecognised format.</exception>
         public static ImageSize GetDimensions(string path, ILogger logger, IFileSystem fileSystem)
         {
-            var extension = Path.GetExtension(path);
+            string extension = Path.GetExtension(path);
```

Comment:
IMO better fix: First check if the path is null or empty, if so throw an ArgumentNullException.
And just `string ect = Path.GetExtension(path).ToLower()`

Reply:
I think getextension can return null if there is no extension, so I safed it

Reply:
I can probably add a throw if the argument is null, but I'd like your comment on my reasoning above ^ :)

Reply:
No, GetExtension only returns null when path is null.

Reply:
My bad, getextension would return empty string in that case. I can just lowercase it. Is it very necessary to add a protection against null arg?

Reply:
Yes, it will make it easier to trace a null error, if there is one.

Reply:
Ok, I'll do this at next opportunity :)